src/components/VmDetails/cards/DisksCard/DiskImageEditor.js

```diff
@@ -423,8 +431,24 @@ class DiskImageEditor extends Component {
                 }
               </Col>
             </FormGroup>
+            {/* Disk Bootable */}
+            { !createMode &&
+              <FormGroup controlId={`${idPrefix}-bootable`}>
+                <LabelCol sm={3}>
+                  { msg.diskEditorBootableLabel() }
+                </LabelCol>
+                <Col sm={9}>
+                  {
```

Comment:
The extra block is not necessary, please remove it and have `Checkbox` be the direct child of `Col`.